wallet/schnapps/res/values/values.xml

```diff
@@ -18,7 +18,7 @@
 		<item>µtDASH, no decimal places</item>
 	</string-array>
 	<string-array name="preferences_block_explorer_values">
-		<item>https://insight.chacha.networks.dash.org:3002/insight</item>
+		<item>https://insight.bintang.networks.dash.org:3002/insight</item>
```

Comment:
bintang is dead.  We could probably change this to %s and fill it in based on the actual network used

Reply:
I think this isn't used at all. In constants, we set the URL based on the network name.